TeamCode/src/main/java/org/firstinspires/ftc/teamcode/hardware/Drivetrain.java

```diff
@@ -43,6 +43,16 @@ public Drivetrain(DcMotor front_left, DcMotor front_right, DcMotor back_left, Dc
 
         front_right.setDirection(DcMotorSimple.Direction.REVERSE);
         back_right.setDirection(DcMotorSimple.Direction.REVERSE);
+
+        front_right.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        back_right.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        front_left.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        back_left.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+
+        front_right.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        back_right.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        front_left.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        back_left.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
```

Comment:
Is the intention to use the inbuilt RUN_TO_POSITION?

Reply:
No. I just wanted to access the encoders. Apparently I didn't need the run mode.